### ISSUE TITLE
How to use external text data?

### ISSUE THREAD

**Issue by arjohnson10** [prakashpandey9/Text-Classification-Pytorch#2]
Hello, If I want to load separate external CSV files for training and testing where the first column of the CSV contains sentences and 2nd column contains the labels then where should I change?

**Comment by prakashpandey9** [comment]
@arjohnson10 Hi. You can use **Torchtext** for preprocessing and loading your CSV files. There are lots of tutorials available online. Just google it.

**Comment by arjohnson10** [comment]
Ya, I found many. I am new to Pytorch. Thanks; And also it will be nice if you can introduce weight visualizations for testing.

**Comment by prakashpandey9** [comment]
Sure. I will try to add it.
